sdks/python/apache_beam/runners/worker/data_plane.py

```diff
@@ -303,9 +334,10 @@ def inverse(self):
 
   def input_elements(self,
       instruction_id,  # type: str
-      unused_expected_inputs=None,   # type: Collection[str]
+      unused_expected_inputs,   # type: Any
```

Comment:
This method was incompatible with its super type.  If it is `None` it invalidates the Liskov substitution principle.  

It's more accurate to say `Any` here, since the method doesn't care, and this keeps the method compatible with its super type.   

I checked all uses of `input_elements` and I did not see any case where it was called with only one arg.